### PR TITLE
Add STIS notebook STIS_Coronagraphy_Visualization_v2.ipynb

### PR DESCRIPTION
**Pull request by dulude**
**Relevant Ticket**
- [SPB-1345](https://jira.stsci.edu/browse/SPB-1345)

**Summary**
- Added notebook _STIS_Coronagraphy_Visualization_v2.ipynb_ (migrated from [spacetelescope/STIS-Notebooks@7e45ca6](https://github.com/spacetelescope/STIS-Notebooks/tree/7e45ca6802e4d216307a133a5c825117d385c499)) and associated files
- All previous commit history has been preserved in the migration of this notebook from the [STIS-Notebooks](https://github.com/spacetelescope/STIS-Notebooks) repository to [hst_notebooks](https://github.com/spacetelescope/hst_notebooks) repository
- Created notebook-specific requirements.txt file
- Updated _toc.yml accordingly

### CODE REVIEW

**Review by haticekaratay — APPROVED** [review]
Thanks, Mike! They all look good to me!

**Comment by haticekaratay** [comment]
This pull request was initially approved on the condition that a follow-up pull request would address the Pep8/pycodestyle style guide checks. However, we now require that all checks be included in this pull request. Please only merge once a **second approval** is given.

**Comment by haticekaratay** [comment]
To adhere to the [STScI style-guides](https://github.com/spacetelescope/style-guides/tree/master/guides) for notebooks, we should include the following elements to ensure the message is clear to the readers:

- Start with learning goals at the beginning of the notebook, describing what the reader will accomplish by the end.
- After the introduction, include imports with clear notes explaining the reasons for importing each.
- Consider including a footer with STScI branding at the bottom of the page, as shown in the [example notebook](https://github.com/spacetelescope/style-guides/blob/master/templates/example_notebook.ipynb) on the STScI website.
- Add descriptive **alt** text to all images.
- Describe all visualizations, including type, title, axis, legend, and an explanation of the visualization's significance to the notebook, such as a trend, an outlier in the data, or something the author learned from it. For more information on making your notebook more accessible, see the [checklist](https://github.com/spacetelescope/jdat_notebooks/blob/main/.github/PULL_REQUEST_TEMPLATE.md).
- Define all acronyms, at least the first time they are used, for instance, "STIS.”

File: notebooks/STIS/CoronagraphyViz/STIS_Coronagraphy_Visualization_v2.ipynb [hunk 1]
```diff
@@ -0,0 +1,505 @@
+{
```
Comment (haticekaratay) [inline notebooks/STIS/CoronagraphyViz/STIS_Coronagraphy_Visualization_v2.ipynb:1]: <p>Based on its purpose, this function was designed to fit the requirements of this notebook, even though there may be some parts that appear to be duplicated. My suggestion is to leave it as it currently is. Additionally,&nbsp;the current version is more straightforward for readers who may not be familiar with programming.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/47/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

**Comment by dulude** [comment]
Please fix the PEP8 style issues before merging this PR.

**Comment by haticekaratay** [comment]
I have fixed all the pep8 errors to speed up the technical review process.

**Pull request merged**